app/models/card.rb

```diff
@@ -1,5 +1,5 @@
 class Card < ActiveRecord::Base
-  before_create  :increase_review_date
+  before_create  :initialize_review_date
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.